x/staking/keeper/val_state_change.go

```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/cosmos/cosmos-sdk/x/staking/types"
 )
 
-// Calculate the ValidatorUpdates for the current block
+// BlockValidatorUpdates calculates the ValidatorUpdates for the current block
```

Comment:
I have a rosetta error [here](https://github.com/cosmos/cosmos-sdk/pull/8363/checks?check_run_id=1768854215):

```
Error: reconciliation failure: active reconciliation error for cosmos1wjmt63j4fv9nqda92nsrp2jp2vsukcke4va3pt at 3 (computed: 100stake, live: 0stake)
```

@alessio @jgimeno I spent some time looking at Rosetta operations impl in x/staking, but didn't find anything to fix. How do you generally debug these tests?

Reply:
if addresses changed somehow due to this PR then most likely the derive endpoint needs to be changed too, I'll have a look

Reply:
@amaurymartiny I've fixed the issue, let me know if I can commit here, it doesn't touch your work.

On a second note, it was failing because cosmos-sdk was crashing as the old validator address format was not valid anymore. Is it intentional?

Reply:
> On a second note, it was failing because cosmos-sdk was crashing as the old validator address format was not valid anymore. Is it intentional?

Hmm, validator address format should not change.

Could you instead create a PR whose base is this PR?

Reply:
Already done: https://github.com/cosmos/cosmos-sdk/pull/8453

I will have a look at the current code and paste the crash error.

Reply:
I've added context to the error: https://github.com/cosmos/cosmos-sdk/pull/8363#issuecomment-768297343